typescript/web/src/components/website/Navbar/NavContent.tsx

```diff
@@ -55,7 +55,7 @@ const MobileNavContext = (props: FlexProps) => {
         )}
         <NextLink href="/local/datasets">
           <Button colorScheme="brand" w="full" size="lg" mt="5">
-            Try it now
+            Try it! No login required
```

Comment:
Same remark, text too long for a button. 

In this example https://pro.chakra-ui.com/components/marketing/pricing  they add a little text under the button for small remarks like that.

Reply:
same remark